.github/workflows/sp_change.yml

```diff
@@ -0,0 +1,22 @@
+name: Stored Procedure Change Check
+
+on:
+  pull_request:
+    types: [opened, synchronize, labeled, unlabeled]
+    branches:
+      - main
+    paths:
+      - 'src/snowflake/connector/cpp/**'
+      - 'src/snowflake/connector/*.py*'
+      - '!src/snowflake/connector/*ocsp*'
+      - '!src/snowflake/connector/*telemetry*'
+
+jobs:
+  check_sp_change:
+    runs-on: ubuntu-latest
+    if: ${{!contains(github.event.pull_request.labels.*.name, 'NO-CHANGE-IN-SP')}}
```

Comment:
What happens for changes that need to be ported? Should we check if the comment/description has a PR link of certain format?